pkg/udevevent/monitor.go

```diff
@@ -20,21 +20,40 @@ import (
 	"errors"
 	"syscall"
 
-	"github.com/openebs/node-disk-manager/cmd/ndm_daemonset/controller"
+	"k8s.io/klog"
+
```

Comment:
If we're going to use this package as a library, I am not sure if we should be logging. It makes sense to me because the cmd/ packages can use these libraries, receive an error and log them. Would like to know your thoughts too @kmova @akhilerm

Reply:
Agree with you here.